air/src/lib.rs

```diff
@@ -157,6 +191,21 @@ impl Air for ProcessorAir {
         );
     }
 
+    fn evaluate_aux_transition<F, E>(
+        &self,
+        main_frame: &EvaluationFrame<F>,
+        aux_frame: &EvaluationFrame<E>,
+        _periodic_values: &[F],
+        aux_rand_elements: &winter_air::AuxTraceRandElements<E>,
+        result: &mut [E],
+    ) where
+        F: FieldElement<BaseField = Self::BaseField> + From<Felt>,
+        E: FieldElement<BaseField = Self::BaseField> + ExtensionOf<F>,
```

Comment:
Would the following work here?:
```
F: FieldElement<BaseField = Felt>,
E: FieldElement<BaseField = Felt>,
```

Reply:
`ExtensionOf` is still required, but `From` isn't. Sorry I overlooked that after covering this in the last PR (it was done this way in the example, but I'll catch it & update next time)